demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1213,6 +1214,27 @@ def is_valid_parameters_display_name(self) -> bool:
 
         return True
 
+    @error_codes('IN153')
+    def is_valid_parameter_url_default_value(self) -> bool:
+        """Verifies integration parameters default value is valid.
+
+        Returns:
+            bool: True if description is valid - capitalized and spaced using whitespace and not underscores,
+            False otherwise.
+        """
+        configuration = self.current_file.get('configuration', {})
+        parameters_default_values = [(param, param.get('defaultvalue')) for param in configuration if param.get('defaultvalue')]
+
+        is_valid = True
+        for param, defaultvalue in parameters_default_values:
+            if defaultvalue and isinstance(defaultvalue, str):
+                if defaultvalue.startswith('http:'):
+                    error_message, error_code = Errors.not_supported_integration_parameter_url_defaultvalue(param, defaultvalue)
+                    if self.handle_error(error_message, error_code, file_path=self.file_path):
+                        is_valid = False
+
+        return is_valid
```

Comment:
```suggestion
        for param, defaultvalue in parameters_default_values:
            if defaultvalue and isinstance(defaultvalue, str):
                if defaultvalue.startswith('http:'):
                    error_message, error_code = Errors.not_supported_integration_parameter_url_defaultvalue(param, defaultvalue)
                    if self.handle_error(error_message, error_code, file_path=self.file_path):
                        return False
        return True
```

Reply:
Can there be multiple parameters with a url that needs to be checked?